source/advanced_components/ddc/usage.rst

```diff
@@ -335,6 +335,54 @@ The main template extends the datastores and network with one EC2 host:
 
 Check the :ref:`Templates <ddc_provision_templates>` reference for available base templates.
 
+.. _ddc_usage_example7:
+
+Example 7: Using more than one playbook
+---------------------------------------
+
+In order to configure the provision, you can specify some playbooks, these are Ansible playbooks that are going to be triggered. You can specify more than one playbook, they are going to be executed one by one by ``oneprovision``.
+
+In the following example we use the default and the static vxlan playbooks:
```

Comment:
``default`` and ``static_vxlan``

(although the combinatino of these 2 doesn't make much sense)